src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.util.Scanner;
+import seedu.tojava.Duke.*;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
 public class Duke {
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException{
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+    }
+
+    public static void showList(ArrayList<Todo> items, int x) {
+        String file1 = "C:\\Users\\Demons\\IdeaProjects\\ip\\out\\duke1.txt";
+
+        if (x == 0) System.out.println("\n____________________________________________________________");
+        else {
+            System.out.println("\n____________________________________________________________");
```

Comment:
Consider adding a line break after this statement to separate from the below "for" statement.

Also, consider abstracting the greeting messages into additional extracted methods.